### PR TITLE
Change timestamp UG, format

### PR DESCRIPTION
**Pull request by koh-jx**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ use via a CLI while still having the benefits of a GUI.
 
 - Parameters must be provided exactly
   
-    - If a parameter is expected only once in the command, but you specified it multiple times,
+    - If a parameter is expected only once in the command. If you specified it multiple times,
```
Comment (jeffsieu) [inline docs/UserGuide.md:75]: Remove the first "If"

**Review by jeffsieu — CHANGES_REQUESTED** [review]
I believe the GUI also needs to change to display in `dd-mm-yyyy`

**Comment by koh-jx** [comment]
> I believe the GUI also needs to be changed to display in `dd-mm-yyyy`. But that could be done in a different PR.

This is done in the `Timestamp#toString()`

**Comment by jeffsieu** [comment]
@koh-jx Conflicts need to be resolved.

**Review by wlren — APPROVED** [review]

**Review by jeffsieu — APPROVED** [review]

**Pull request merged**